master/internal/experiment/authz_iface.go

```diff
@@ -53,6 +53,7 @@ type ExperimentAuthZ interface {
 
 	// GET /api/v1/experiments
 	// "proj" being nil indicates getting experiments from all projects.
+	// WARN: query is expected to expose the "workspace_id" column.
```

Comment:
we should try to add better guards for this than a comment but we're limiting the changes here for this fix.